common/retry_backoff_config.go

```diff
@@ -0,0 +1,66 @@
+package common
+
+import (
+	"fmt"
+	"math"
+	"time"
+
+	commontypes "github.com/agglayer/aggkit/common/types"
+	"github.com/agglayer/aggkit/config/types"
+)
+
+var _ commontypes.RetryPolicyConfigurer = (*RetryBackoffConfig)(nil)
+
+type RetryBackoffConfig struct {
```

Comment:
I'd suggest the rename, in order not to confuse it with the struct that is used to unmarshal the TOML config. WDYT?

```suggestion
type RetryBackoffPolicy struct {
```

Reply:
In fact is a config, not a Policy. The logic is in another object... maybe we can came up with a better name...

Reply:
We discuss it, and both are in the fence, that have a part of config and a part of logic, so there are no clear answer so we agree to left as `...Config`